system-tests/lib/cre/contracts/contracts.go

```diff
@@ -168,30 +170,63 @@ func ConfigureKeystone(input cre.ConfigureKeystoneInput, capabilityRegistryConfi
 	if addrErr != nil {
 		return errors.Wrap(addrErr, "failed to get addresses from address book")
 	}
-	chainsWithForwarders := make(map[uint64]struct{})
+
+	evmChainsWithForwarders := make(map[uint64]struct{})
 	for chainSelector, addresses := range allAddresses {
 		for _, typeAndVersion := range addresses {
 			if typeAndVersion.Type == keystone_changeset.KeystoneForwarder {
-				chainsWithForwarders[chainSelector] = struct{}{}
+				evmChainsWithForwarders[chainSelector] = struct{}{}
 			}
 		}
 	}
 
-	_, err = operations.ExecuteSequence(
-		input.CldEnv.OperationsBundle,
-		ks_contracts_op.ConfigureForwardersSeq,
-		ks_contracts_op.ConfigureForwardersSeqDeps{
-			Env:      input.CldEnv,
-			Registry: capReg.Contract,
-		},
-		ks_contracts_op.ConfigureForwardersSeqInput{
-			RegistryChainSel: input.ChainSelector,
-			DONs:             configDONs,
-			Chains:           chainsWithForwarders,
-		},
-	)
-	if err != nil {
-		return errors.Wrap(err, "failed to configure forwarders")
+	solForwarders := input.CldEnv.DataStore.Addresses().Filter(datastore.AddressRefByQualifier("test-forwarder"))
```

Comment:
is there some constant in the `deployment`/CLD we could use instead of that string literal?